src/modelFactory/FindInitialValues.h

```diff
@@ -26,20 +25,18 @@ namespace DESCAM {
 class FindInitialValues : public IFindInitialValues, public clang::RecursiveASTVisitor<FindInitialValues> {
  public:
   ~FindInitialValues() override = default;
-  FindInitialValues();
+  explicit FindInitialValues(IFindDataFlowFactory *find_data_flow_factory);
 
-  void setup(clang::CXXRecordDecl *recordDecl,
+  bool setup(clang::CXXRecordDecl *recordDecl,
              clang::FieldDecl *fieldDecl,
              DESCAM::Module *module,
-             clang::CompilerInstance *ci,
-             IFindDataFlowFactory * find_data_flow_factory) override;
+             clang::CompilerInstance *ci) override;
 
   ConstValue *getInitValue() override;
   //Visitor
   bool VisitCXXConstructorDecl(clang::CXXConstructorDecl *constructorDecl);
  private:
-  void clean();
```

Comment:
rename "clean()" to "clear()" to emulate std::container conventions, e.g., std::vector::clear()

Reply:
i completely removed the clean()/clear() function because it was not needed.